docs/ibm-cp4na-log-message.md

```diff
@@ -16,13 +16,40 @@ The module will accept the following parameters:
 | --- | --- | --- | --- | 
 | message_direction | str | the `message_direction` to be included in the logs | Yes |
 | message_type | str | the `message_type` to be included in the logs | Yes |
-| message_data | str | the `message_data` to be included in the logs | No |
+| message_data | str | the `message` to be included in the logs | No |
 | content_type | str | the `content_type` to be included in the logs | Yes if `message_data` is provided |
 | protocol | str | the `protocol` to be included in the logs | Yes |
 | protocol_metadata | dict | the `protocol_metadata` to be included in the logs (will be formatted as a JSON string by the module/driver) | No |
 | external_request_id | String | the `tracectx.externalrequestid` to be included in the logs | Yes | 
 
-**Note**: supported values for `message_direction` is : `sent` or `received`
+**Notes**: 
+  1. Supported values for `message_direction` are : `sent` and `received`.
+  2. Supported values for `message_type` are : `request`, `response` and `message`.
+  3. Supported values for `protocol` are : `http`, `cmd` or any other protocols.
```

Comment:
I'd suggest changing this to: 

3. The `protocol` value is user defined based on the type of request/response you intend to log. However, please note that `http` and `cmd` are known protocols in Network Automation, with suggested `protocol_metadata` attributes and a tailored user experience. See below for more details and also consult the documentation of your Network Automation installation.

Reply:
Ok, I have changed it now.